Sources/MediaNavigationBar.swift

```diff
@@ -57,6 +66,7 @@ protocol MediaNavigationBarDelegate {
         var airPlayRoutePicker = AVRoutePickerView(frame: .zero)
         airPlayRoutePicker.activeTintColor = .orange
         airPlayRoutePicker.tintColor = .white
+        airPlayRoutePicker.widthAnchor.constraint(equalToConstant: buttonSize).isActive = true
```

Comment:
is this really necessary? don't the buttons resize themself based on the image?

Reply:
Actually despite trying setContentHuggingPriority on the buttons, the minimize and chromecast buttons are unwilling to stay up without the widths set. "Width and horizontal position are ambiguous"